packages/playwright/src/reporters/html.ts

```diff
@@ -86,8 +86,11 @@ class HtmlReporter extends EmptyReporter {
     this._open = open;
     this._attachmentsBaseURL = attachmentsBaseURL;
     const reportedWarnings = new Set<string>();
+    const normalizePath = (path: string) => path.replace(/\\/g, '/').replace(/([^/])$/, '$1/');
```

Comment:
Stack Overflow recommends to use `path.relative()` for this, which sounds more robust than wrangling with the slashes: https://stackoverflow.com/questions/37521893/determine-if-a-path-is-subdirectory-of-another-in-node-js.